libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-table/dh-processes-table.component.html

```diff
@@ -0,0 +1,150 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *ngIf="processes && processes.length > 0; else emptyTab">
+  <mat-table
+    #matTable
+    [dataSource]="sortedData"
+    matSort
+    (matSortChange)="sortData($event)"
+  >
+    <!-- Expand Definition -->
+    <ng-container matColumnDef="expand">
+      <mat-header-cell *matHeaderCellDef class="watt-label column-expand">
+      </mat-header-cell>
+      <mat-cell *matCellDef="let row">
+        <watt-icon name="right" [size]="iconSize.Large"></watt-icon>
+      </mat-cell>
+    </ng-container>
+
+    <!-- Name Definition -->
+    <ng-container matColumnDef="name">
+      <mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        [sortActionDescription]="
+          'meteringPoint.overview.processesTab.sortDescription.name' | transloco
```

Comment:
suggestion: Instead of repeating the `meteringPoint.overview.processesTab` many times, you can set it as a scope on a wrapping element and only use `sortDescription.name` to display the translation. Example `dh-metering-point-primary-master-data.component.html#18`.